bundle/Command/GenerateProjectCommand.php

```diff
@@ -153,11 +154,48 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
         // Various cleanups
         $this->cleanup();
 
+        $this->activateGitHooks();
+
         $this->writeSection(['You can now start using the site!']);
 
         return 0;
     }
 
+    private function activateGitHooks()
+    {
+        if (!class_exists(Config::class)) {
+            return;
+        }
+
+        /** @var \Symfony\Component\Filesystem\Filesystem $fileSystem */
+        $fileSystem = $this->getContainer()->get('filesystem');
+        $projectDir = $this->getContainer()->getParameter('kernel.project_dir');
+        $captainHookFiles = [
+            $projectDir . '/captainhook_disabled.json',
+            $projectDir . '/captainhook_enabled.json',
+        ];
+
+        if ($fileSystem->exists($captainHookFiles)) {
+            return;
+        }
+
+        if ($this->questionHelper->ask(
+                $this->input,
+                $this->output,
+                $this->getConfirmationQuestion(
+                    'Do you want to use default <comment>githooks</comment>?',
```

Comment:
Maybe a better question would be `Do you want to use git hooks suitable for project development?` ? Because in this context, default git hooks are actually the disabled ones (`captainhook_disabled.json`), as they are symlinked by default.

Reply:
Changed in 0f98ae74ccd13a371dfcc04cfa3721fb4212f658